### PR TITLE
Add deployed versions of server, saml2-config, and host-validation

### PR DESCRIPTION
**Pull request by mjoyce91**

### CODE REVIEW

File: src/saml2-config.alt.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  serviceProviderPublic.create_login_request_url(identityProvider, {}, handler);
+};
+
+const acs = (options, handler) => {
+  serviceProvider.post_assert(identityProvider, options, handler);
+};
+
+const logout = (handler) => {
+  serviceProvider.create_logout_request_url(identityProvider, null, handler);
+};
```
Comment (elizabeth-jimenez) [inline src/saml2-config.alt.js:63]: Do we want a `logoutPublic`? If not, how come?

File: src/server-alt.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    'Cache-Control': 'public',
+  });
+  res.removeHeader('Surrogate-Control');
+  res.removeHeader('Pragma');
+  res.removeHeader('Expires');
+  next();
+};
+// define full path to static build
+const STATIC_PATH = process.env.STATIC_PATH || path.join(__dirname, '../build');
+// define the tPI root url
```
Comment (elizabeth-jimenez) [inline src/server-alt.js:22]: comment typo?

File: src/saml2-config.alt.js (hunk 1 above)
Comment (mjoyce91) [reply src/saml2-config.alt.js:63]: Slightly different in State department. Our AWS setup is to use either Persona Login OR ADFS login. But in DOS it's all ADFS, whether that's on OpenNET or Go Browser

Comment (elizabeth-jimenez) [reply src/saml2-config.alt.js:63]: Thanks!

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 👍

**Pull request merged**